src/features/events/components/EventOverviewCard/index.tsx

```diff
@@ -80,7 +76,7 @@ const EventOverviewCard: FC<EventOverviewCardProps> = ({
         setShowEndDate(false);
       },
       save: () => {
-        dataModel.updateEventData({
+        updateEvent(data.id, {
```

Comment:
ID should not be necessary here (if it was passed to `useEventMutations()` instead.